tests/notification.spec.ts

```diff
@@ -1,40 +1,60 @@
-import { test, expect } from '@playwright/test';
+import { test, expect, request } from '@playwright/test';
 import { api } from './utils';
+import config from '../init/config';
 
-let ideaId = 0;
-
-test.describe('/api/notification', () => {
-
-    test("Create new notification on idea create", async ({ request }) => {
-        const newIdea = await api(request).post("/api/idea-cards", {
-            ideaName: "Testing2",
-            tagline: "Yay!",
-            description: "I am testing",
-            targetAudience: "Yes",
-            features: "Yes",
-            experience: "Yes",
-            extraInfo: "Yes",
-            involveLevel: "minimum",
-            status: "workshopping",
+let user1Jwt, user2Jwt, notificationId;
+
+test.describe('/api/notification policy', () => {
+    test.beforeAll(async ({ request }) => {
+        const apiContext = api(request);
+
+        // Authenticate user1
+        const user1LoginResponse = await apiContext.post('/api/auth/local', {
```

Comment:
So, instead of logging in two users we can check it for the same user. 
- we can create two notifications with two different users ids.
- then when we read notifications by currently logged in user, user should see only one notification
- to create notification we should use Strapi service, not POST /api/notifications. POST to /api/notification is disabled. You can grab Strapi object using `const strapi = await strapiConnect();` from https://github.com/dev-launchers/strapiv4/blob/main/tests/utils.ts#L60